### ISSUE TITLE
Fetch m3u8 for movies

### ISSUE THREAD

**Issue by Rhiannon99** [ZeroSkillSamus/Video-Extractor-API#1]
Hello brother how i can fetch movies and please can u add embed url to beside m3u8 urls so i can store all movies with imdb id then use the extractor thank you

**Comment by ZeroSkillSamus** [comment]
I am working on the Readme now, but here is what the routes are

The IDs are from watchseries 
```
/VIDSRC-TO/Watch?id=<str>&season=<int>&episode=<int> -> for movies season and episode need to be 1
/WATCHSERIES/Search?query=<str> -> query (whatever you want to search)
/WATCHSERIES/Trending 
/WATCHSERIES/Details?id=<str> -> needs id
```

**Comment by Rhiannon99** [comment]
Thank u bro one more question possible to fetch embed urls for filemoon and f2cloud

**Comment by ZeroSkillSamus** [comment]
Sure I will get on that for u

**Comment by itsallan** [comment]
@ZeroSkillSamus  am sorry if am I may sound pressuring but when should we expect the documentation but I have failed to make the API work even fetching trending movies

**Comment by ZeroSkillSamus** [comment]
Really? I will do the documentation now then but is there an error you are getting?

**Comment by itsallan** [comment]
Yeah I folked the repo and then created a new project on vercel them selected it from GitHub but all am receiving is this 
![Screenshot_20240810-004516](https://github.com/user-attachments/assets/93d89ef4-d816-4bf9-8e29-4fb4a7655aa4)
   This is the link https://video-extractor-api-six.vercel.app/
